gotool/gotool.go

```diff
@@ -0,0 +1,44 @@
+package gotool
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/command"
+	glob "github.com/ryanuber/go-glob"
+)
+
+func parsePackages(out string, exclude ...string) []string {
+	list := []string{}
+	for _, l := range strings.Split(string(out), "\n") {
+		l = strings.TrimSpace(l)
+		if l == "" {
+			continue
+		}
+
+		match := false
+		for _, e := range exclude {
+			if glob.Glob(e, l) {
+				match = true
+				break
+			}
+		}
+		if match {
+			continue
+		}
+
+		list = append(list, l)
+	}
+	return list
+}
+
+// ListPackages ...
+func ListPackages(dir string, exclude ...string) ([]string, error) {
```

Comment:
it is hard to understand that you want to run a function that expects a dir but you set not the dir itself, just the command's working dir.